prep-html-links/contact-us/contact-us.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Contact Us</title>
+</head>
+<body>
+  <h1>Contact Us</h1>
+<a href="http://127.0.0.1:5500/index.html" target="_blank">Click here to go to the home page</a>
```

Comment:
Here you'll want to use the `..` syntax for making a relative URL in your `href` attribute.

```html
href="../index.html"
```

Otherwise, this link won't work properly unless someone else is using Live Server.

Reply:
Updated, Thank You!